modin/experimental/core/execution/native/implementations/omnisci_on_native/test/utils.py

```diff
@@ -88,21 +90,52 @@ def align_datetime_dtypes(*dfs):
     Passed Modin frames may be casted to pandas in the result.
     """
     datetime_cols = {}
+    time_cols = set()
     for df in dfs:
         for col, dtype in df.dtypes.items():
             # If we already decided to cast this column to DateTime no more actions are needed
             if col not in datetime_cols and is_datetime64_any_dtype(dtype):
                 datetime_cols[col] = dtype
-
-    casted_dfs = (
+            # datetime.time is considered to be an 'object' dtype in pandas that's why
+            # we have to explicitly check the values type in the column
+            elif (
+                dtype == np.dtype("O")
+                and col not in time_cols
+                # OmniSci has difficulties with empty frames, so explicitly skip them
+                # https://github.com/modin-project/modin/issues/3428
```

Comment:
The issue was closed. Does it still persists?

Reply:
I still encounter problems with empty frames, I've updated the linked issue ([comment](https://github.com/modin-project/modin/issues/3428#issuecomment-1014443549))

Reply:
Ok, thanks!